mastering-plone/dexterity_2.rst

```diff
@@ -101,7 +110,8 @@ Then we can safely bind the ``talkview`` to the new marker interface.
         permission="zope2.View"
         />
 
-Now the ``/talkview`` can only be used on objects that implement said interface. We can now also query the catalog for objects providing this interface :py:meth:`catalog(object_provides="ploneconf.site.interfaces.ITalk")`. The ``talklistview`` and the ``demoview`` do not get this constraint since they are not only used on talks.
+Now the ``/talkview`` can only be used on objects that implement said interface. We can now also query the catalog for objects providing this interface :py:meth:`catalog(object_provides="ploneconf.site.interfaces.ITalk")`.
```

Comment:
Does this render correctly? I haven't verified, but it looks unusual to me.

```
:py:meth:`catalog(object_provides="ploneconf.site.interfaces.ITalk")`
```

Reply:
no idea. it was that way, I kept it so